app/react/Attachments/components/Attachment.js

```diff
@@ -149,7 +198,53 @@ export class Attachment extends Component {
     return (
       <div className="attachment">
         {name}
-        {buttons}
+        <NeedAuthorization roles={['admin', 'editor']}>
+          {buttons}
+
+          <div className="dropdown attachments-dropdown">
+            <button
+              className="btn btn-default dropdown-toggle attachments-dropdown-toggle"
+              type="button"
+              id="attachment-dropdown-actions"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              onClick={this.toggleDropdown}
+            >
+              <Icon icon="pencil-alt" />
+            </button>
+            <ul
+              className="dropdown-menu dropdown-menu-right"
+              aria-labelledby="attachment-dropdown-actions"
+              style={{ display: this.state.dropdownMenuOpen ? 'block' : 'none' }}
+              ref={this.myRef}
+            >
+              <li>
+                <button type="button" onClick={e => this.copyToClipboard(item)}>
```

Comment:
here is an eslint error. we should underscore the `e`, or maybe not even declare it.